src/components/Sidebar.tsx

```diff
@@ -309,6 +309,27 @@ export default function Sidebar(props: Props) {
                     }>
                     {constants.LOGOUT}
                 </LinkButton>
+                <LinkButton
+                    variant="danger"
+                    style={{ marginTop: '30px' }}
+                    onClick={() =>
+                        props.setDialogMessage({
+                            title: `${constants.DELETE_ACCOUNT}`,
+                            content: constants.DELETE_MESSAGE(),
+                            staticBackdrop: true,
+                            proceed: {
+                                text: constants.DELETE_ACCOUNT,
+                                action: () => {
```

Comment:
you can create a function for a action and call it here. 

Something similar to https://github.com/ente-io/bada-frame/blob/1d2b6e3243b4e40be548db30446fe539e3b627c5/src/components/Sidebar.tsx#L79

Reply:
Done, thanks for this suggestion.